### PR TITLE
2.0.0-rc

### PR DESCRIPTION
**Pull request by sezanzeb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,11 +60,20 @@ sudo systemctl enable input-remapper
 sudo systemctl restart input-remapper
 ```
 
-If it doesn't seem to install, you can also try `sudo python3 setup.py install`
+## Migrating old configs to version 2
```
Comment (jonasBoss) [inline README.md:63]:
```suggestion
## Migrating beta configs to version 2
```
Since this section only concerns beta configs.

File: README.md [hunk 2]
```diff
@@ -58,11 +60,20 @@ sudo systemctl enable input-remapper
...
+
+If you never installed beta alongside v1, there is no need to do anything.
+
+If you want to migrate your beta configs to v2, run
```
Comment (jonasBoss) [inline README.md:67]:
```suggestion

By default Inputremapper will only migrate configurations from the v1 main branch.
If you want to use configurations created on the beta branch you will need to copy 
them manually. Make sure the `~/.config/input-remapper-2` folder is empty before 
you do so.
```

File: README.md [hunk 3]
```diff
@@ -58,11 +60,20 @@ sudo systemctl enable input-remapper
...
+If you want to migrate your v1 configs to v2, run
 
-The `beta` branch contains features that still require work, but that are ready for testing. It uses a different
-config path, so your presets won't break. `input-remapper-beta-git` can be installed from the AUR. If you are
-facing problems, please open up an [issue](https://github.com/sezanzeb/input-remapper/issues).
+```
+cp ~/.config/input-remapper/ ~/.config/input-remapper-2 -r
+```
 
+Then start input-remapper
```
Comment (jonasBoss) [inline README.md:79]: This is confusing, I thought Inputremapper would do this by default.

**Comment by sezanzeb** [comment]
It's ready to merge this into main.

There is just some minor stuff remaining, that can also be done on the main branch before the proper release

**Review by jonasBoss — APPROVED** [review]

**Pull request merged**